primitives/src/lib.rs

```diff
@@ -508,6 +508,17 @@ pub struct CustomMetadata {
 }
 
 impl CurrencyId {
+    pub fn sort(&mut self) {
```

Comment:
```suggestion
    pub fn canonicalize(&mut self) {
```
Calling sort on a single item is a bit weird so I think this naming makes more sense

Reply:
Sticking to Uniswap naming: https://docs.uniswap.org/contracts/v2/reference/smart-contracts/library#sorttokens

Reply:
See also: https://github.com/interlay/interbtc/blob/1f7533459a654d745002d1e519cf4fe9ecc4dfe2/crates/dex-general/src/swap/mod.rs#L34

Reply:
That's on multiple inputs whereas you implement it on a single CurrencyId. Imo `canonicalize` makes a lot more sense but I won't block on this